storage/src/test/java/zipkin2/storage/kafka/KafkaStorageIT.java

```diff
@@ -49,21 +51,23 @@
 import zipkin2.storage.kafka.streams.serdes.SpansSerde;
 
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.awaitility.Awaitility.await;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
 
 @Testcontainers
 class KafkaStorageIT {
-  private static final long TODAY = System.currentTimeMillis();
+  static final Logger LOG = LogManager.getLogger();
 
-  @Container private KafkaContainer kafka = new KafkaContainer("5.3.0");
+  static final long TODAY = System.currentTimeMillis();
 
-  private Duration traceTimeout;
-  private KafkaStorage storage;
-  private Properties consumerConfig;
-  private KafkaProducer<String, List<Span>> tracesProducer;
-  private KafkaProducer<String, DependencyLink> dependencyProducer;
+  // TODO: does this need to be confluent container?
```

Comment:
@adriancole nope, it is just the one used by testcontainers https://www.testcontainers.org/modules/kafka/

we could use our own via generic containers tho, wdyt?

Reply:
yeah if we use ours it will reduce questions I think at least in our own group